plugins/omar-wms-plugin/grails-app/services/omar/wms/WebMappingService.groovy

```diff
@@ -268,11 +285,16 @@ class WebMappingService implements InitializingBean
 
   def getMap(GetMapRequest wmsParams)
   {
-    def otherParams = [startDate: new Date()]
+    def startTime = new Date()
+    def internalTime = new Date()
+    def procTime = new Date()
 
     log.trace "getMap: Entered ................"
-    otherParams.startTime = System.currentTimeMillis()
-    otherParams.internalTime = otherParams.startTime
+    startTime = System.currentTimeMillis()
+    internalTime = otherParams.startTime
+
+    log.info("getMap timestamp " + new Date().format("YYYY-MM-DD HH:mm:ss.Ms"))
```

Comment:
You can use Groovy's String interpolation to build the Date object and call format() on it within the "".

log.info("getMap timestamp ${new Date().format("YYYY-MM-DD HH:mm:ss.Ms")}")